authCommon/app/modules/dao/IssuerNotFoundError.js

```diff
@@ -0,0 +1,8 @@
+class IssuerNotFoundError extends Error {
```

Comment:
Perche questo non estende AuthenticationError?

Reply:
@marco-vit-pagopa per due motivi:
- tenere confinata l'area di responsabilità del DAO, visto che è lui che solleva l'eccezione; preferisco che sia la logica di business dell'authorizer a decidere se quell'eccezione DAO debba tradursi in un AuthenticationError
- la classe AuthenticationError non è definita nel package comune auth-common (questa è una mini-minor reason)

Reply:
Si giusto, hai ragionissimo.